src/components/tag/Tag.props.ts

```diff
@@ -0,0 +1,13 @@
+import { TTagVariant, TTagSizeToken } from './Tag.types';
+
+type PTag = {
+    variant: TTagVariant;
+    text: string;
+    size?: TTagSizeToken;
+    onClick?: () => void;
+    className?: string;
+};
```

Comment:
please provide descriptions for the props

FYI: the description will get ported to the `PTagRoot` as well, when using utility types! :D